src/server/scripts/Spells/spell_paladin.cpp

```diff
@@ -136,6 +136,12 @@ enum PaladinSpellIcons
     PALADIN_ICON_ID_HAMMER_OF_THE_RIGHTEOUS      = 3023
 };
 
+enum MiscSpellIcons
+{
+    SPELL_ICON_ID_STRENGTH_OF_WRYNN                 = 1704,
+    SPELL_ICON_ID_HELLSCREAM_WARSONG                = 937
```

Comment:
Cosmetic / codestyle: when everything else in this PR has been reviewed and verified OK, align these 2 enum value indents with the previous enum indents.